.github/workflows/build.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         run: composer test
 
       - name: Check dependencies for known security vulnerabilities
-        run: composer audit
+        run: composer audit --no-dev
```

Comment:
How is this related?

Reply:
Proof of failure: https://github.com/php-tuf/php-tuf/actions/runs/8266701537/job/22615313427#step:13:1